### PR TITLE
Enable and fix Journalbeat integration tests

### PR DESCRIPTION
**Pull request by kvch**

### CODE REVIEW

**Comment by kvch** [comment]
jenkins run tests

**Comment by kvch** [comment]
jenkins run tests

File: journalbeat/tests/system/test_base.py [hunk 1]
```diff
@@ -74,17 +69,7 @@ def test_start_with_selected_journal_file(self):
...
-            "journalbeat is running",
-            # journalbeat can seek to the position defined in the cursor
-            "Reading from the beginning of the journal file",
-            # message can be read from test journal
-            "\"message\": \"thinkpad_acpi: unhandled HKEY event 0x60b0\"",
-        ]
-        for snippet in required_log_snippets:
-            self.wait_until(lambda: self.log_contains(snippet),
-                            name="Line in '{}' Journalbeat log".format(snippet))
+        self.wait_until(lambda: self.output_has(lines=23))
```
Comment (mdelapenya) [inline journalbeat/tests/system/test_base.py:72]: I noticed this assertion is different to any of the former ones, and besides that, the changes in other tests are asserting that the output contains `"journalbeat is running"`, is this on purpose?

File: journalbeat/tests/system/test_base.py [hunk 2]
```diff
@@ -143,19 +121,7 @@ def test_read_events_with_existing_registry(self):
...
-            "Seeked to position defined in cursor",
-            # message can be read from test journal
-            "please report the conditions when this event happened to",
-            # only one event is read and published
-            'journalbeat successfully published events\t{"event.count": 1}',
-        ]
-        for snippet in required_log_snippets:
-            self.wait_until(lambda: self.log_contains(snippet),
-                            name="Line in '{}' Journalbeat log".format(snippet))
+        self.wait_until(lambda: self.output_has(lines=9))
```
Comment (mdelapenya) [inline journalbeat/tests/system/test_base.py:124]: Same here.

Once more thing: how likely is this output to not contain `9` lines?

File: journalbeat/tests/system/test_base.py [hunk 3]
```diff
@@ -173,27 +139,13 @@ def test_read_events_with_include_matches(self):
...
-            "unhandled HKEY event 0x60b0",
-            "please report the conditions when this event happened to",
-            "unhandled HKEY event 0x60b1",
-            # Four events with priority 5 is publised
-            'journalbeat successfully published events\t{"event.count": 4}',
-        ]
-        for snippet in required_log_snippets:
-            self.wait_until(lambda: self.log_contains(snippet),
-                            name="Line in '{}' Journalbeat log".format(snippet))
+        self.wait_until(lambda: self.output_has(lines=6))
```
Comment (mdelapenya) [inline journalbeat/tests/system/test_base.py:148]: Same here

**Pull request merged**